app/components/task_list_items/cil_liability_component.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  class CilLiabilityComponent < TaskListItems::BaseComponent
+    def initialize(planning_application:)
+      @planning_application = planning_application
+    end
+
+    private
+
+    attr_reader :planning_application
+
+    delegate(:cil_liable, to: :planning_application)
+
+    def link_text
+      "CIL Liability"
+    end
+
+    def link_path
+      edit_planning_application_cil_liability_path(
+        planning_application
+      )
+    end
+
+    def status
+      if cil_liable.nil?
+        :not_started
+      else
+        "cil_liable_#{cil_liable}".to_sym
+      end
+    end
+
+    def to_be_reviewed?
+      planning_application.recommendation&.rejected? &&
+        permitted_development_right.update_required?
```

Comment:
I notice there's a few of these `&.` for the `recommendation` across the application - fine to leave this for now but something to revisit everywhere on another day as it's not good OOP pattern to have business logic in a view component.